neutrons/addie
# Added missing 'import pyoncat' calls

Fixes #229 

*To Test*:
 - have to uninstall pyoncat (I did `pipenv uninstall pyoncat`)
 - launch `addie`
 - check that you do not have the option for Import from Database in the right-click menu

To revert the pyoncat change, I just did `git checkout -- Pipfile; pipenv install`
